crates/nu-command/src/filters/merge/deep.rs

```diff
@@ -39,8 +41,64 @@ impl Command for MergeDeep {
     }
 
     fn examples(&self) -> Vec<Example> {
-        // TODO(rose)
-        vec![]
+        vec![
+            Example {
+                example: "{a: 1, b: {c: 2}} | merge deep {b: {c: 3, d: 4}}",
+                description: "Merge two records",
+                result: Some(Value::test_record(record! {
+                    "a" => Value::test_int(1),
+                    "b" => Value::test_record(record! {
+                        "c" => Value::test_int(3),
+                        "d" => Value::test_int(4),
+                    })
+                })),
```

Comment:
```suggestion
                example: "{a: 1, b: {c: 2}} | merge deep {b: {d: 4, e: 5}}",
                description: "Merge two records recursively",
                result: Some(Value::test_record(record! {
                    "a" => Value::test_int(1),
                    "b" => Value::test_record(record! {
                        "c" => Value::test_int(3),
                        "d" => Value::test_int(4),
                        "e" => Value::test_int(5),
                    })
                })),
```
this is technically the same as what `merge` would do, so add an extra value here to differentiate

Reply:
Good catch, even with you pointing it out, it took me minute to realize that it could be misleading.